lib/ContextReplacementPlugin.js

```diff
@@ -66,7 +66,7 @@ class ContextReplacementPlugin {
 				}
 				return callback(null, result);
 			});
-			cmf.plugin("after-resolve", (result, callback) => {
+			cmf.hooks.afterResolve.tapAsync("ContextReplacementPlugin", (result, callback) => {
```

Comment:
Use `tap` instead of `tapAsync` and remove the `callback`. Return the result instead.